doc/api/net.markdown

```diff
@@ -117,22 +94,23 @@ changed to
 ## net.connect(port[, host][, connectListener])
 ## net.createConnection(port[, host][, connectListener])
 
-Creates a TCP connection to `port` on `host`. If `host` is omitted,
-`'localhost'` will be assumed.
+A factory function, which returns a new ['net.Socket'](#net_class_net_socket)
+and automatically connects to the supplied `port` and `host`.
+
+If host is omitted, 'localhost' will be assumed.
+
 The `connectListener` parameter will be added as an listener for the
 ['connect'][] event.
 
-Is a factory method which returns a new ['net.Socket'](#net_class_net_socket).
-
 ## net.connect(path[, connectListener])
 ## net.createConnection(path[, connectListener])
 
-Creates unix socket connection to `path`.
+A factory function, which returns a new unix ['net.Socket'](#net_class_net_socket)
```

Comment:
Yes. Is not a big problem that some lines are a little short. It will be at same line in the rendered version anyway.